packages/account/src/Components/scrollbars-container/scrollbars-container.tsx

```diff
@@ -1,24 +1,33 @@
 import classNames from 'classnames';
 import React from 'react';
 import { ThemedScrollbars } from '@deriv/components';
-import { isMobile } from '@deriv/shared';
+import { useDevice } from '@deriv-com/ui';
 
 type TScrollbarsContainer = {
     className?: string;
     scroll_offset?: string;
+    isFullHeight?: boolean;
 };
 
 export const ScrollbarsContainer = ({
     children,
     className,
     scroll_offset,
-}: React.PropsWithChildren<TScrollbarsContainer>) => (
-    <ThemedScrollbars is_bypassed={isMobile()} height={scroll_offset ? `calc(100% - ${scroll_offset})` : '100%'}>
-        <div
-            className={classNames('account__scrollbars_container', className)}
-            data-testid='dt_scrollbar_container_div'
+    isFullHeight = false,
+}: React.PropsWithChildren<TScrollbarsContainer>) => {
+    const { isDesktop } = useDevice();
+    const height_unit = isFullHeight ? '100vh' : '100%';
+    return (
+        <ThemedScrollbars
```

Comment:
This change will impact all components using <FormBody> 👀 . Please ensure they are tested

Reply:
Hi @likhith-deriv your concern is valid but as I made default value false so it should not break anything 😬 
But still I will check other screens as well. 

Reason why I added this. 

Without adding this 100vh I am not able to scroll till down properly on mobile and tablet screen.